test.js

```diff
@@ -69,6 +71,18 @@ test('do not include `stderr` in errors when set to `inherit`', async t => {
 	t.notRegex(err.message, /stderr/);
 });
 
+test('pass `stdout` to a file descriptor', async t => {
+	const file = tempfile('.txt');
+	await m('fixtures/noop', ['foo bar'], {stdout: fs.openSync(file, 'w')});
+	t.is(fs.readFileSync(file, 'utf8'), 'foo bar\n');
+});
+
+test('pass `stderr` to a file descriptor', async t => {
+	const file = tempfile('.txt');
+	await m('fixtures/noop-err', ['foo bar'], {stderr: fs.openSync(file, 'w')});
+	t.is(fs.readFileSync(file, 'utf8'), 'foo bar\n');
```

Comment:
Ah. Good test strategy. File streams are allowed. I opened an issue to handle userland streams better